packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -344,10 +345,10 @@ export default function ProjectNav({
             }`}
           >
             {!isMobile && (
-              <Icon
+              <Atlas
+                active={activeProduct === Product.Cloud && !!current}
```

Comment:
Do we need the double `!!` or not? Seems to be inconsistently applied.

Reply:
We definitely do for the project nav IconButtons where it wasn't previously applied, but actually not sure if we need to disable these tabs in the loading state. I'll talk to Nelle and see

Reply:
Confirmed this logic is correctly applied everywhere :D